### ISSUE TITLE
Relax constraints on brands

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#1077]
Like https://github.com/phetsims/chipper/issues/1076, we would like to relax the constraints on BRANDS so that clients can use a new brand.  I'll take the lead.

**Comment by samreid** [comment]
Fixed, and @jessegreenberg volunteered to review.

**Comment by samreid** [comment]
I tested by copying brands/adapted-from-phet to brands/testbrand, then adding "testbrand" to "brands" in gravity-and-orbits/package.json.  Then running `grunt --brands=testbrand` in gravity-and-orbits/.

**Comment by jessegreenberg** [comment]
Thanks @samreid, looks good. I tested by following the instructions at https://github.com/phetsims/brand/blob/master/README.md. Things mostly worked except for testing the unbuild version I hit
>Invalid value supplied for key "brand": my-brand is not a member of valid values: phet, phet-io, adapted-from-phet

Can we remove the list of `validValues` from initialize-globals?

I also didn't see anything in the README about editing package.json, so I added a note about that. Back to @samreid to review and comment.
